bin/server.py

```diff
@@ -93,8 +92,6 @@ async def init(self, request, name, target_device_name, distro_name, build_type_
         self._init_mongodb()
         self._collection_name = distro_name
         self._collection = self._db[self._collection_name]
-
```

Comment:
Do not delete this blank line. It separates the mongo-related lines from the packages-related ones.